### PR TITLE
i18n(ko-KR): create `code.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description

- create `code.mdx`

### CODE REVIEW

File: docs/src/content/docs/ko/components/code.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Code
+	slot="preview"
+	code={`## 환영합니다
+
+안녕하세요 (**우주에서**)!`}
```
Comment (HiDeoo) [inline docs/src/content/docs/ko/components/code.mdx:19]: Just to double check, are the `(` and `)` around `**우주에서**` expected?

**Review by HiDeoo — COMMENTED** [review]
Thanks for updating the translation :star2:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left one tiny questions but other than that, everything looks good to me.

File: docs/src/content/docs/ko/components/code.mdx (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/components/code.mdx]: I've updated it. Thank you

**Comment by jsparkdev** [comment]
@HiDeoo I've updated it. Thank you

**Review by HiDeoo — APPROVED** [review]
Wonderful work with all these translations! Thanks a lot for all your contributions :raised_hands: 

One last note in case you're interested, the sidebar group label could be translated [here](https://github.com/withastro/starlight/blob/main/docs/astro.config.mjs#L136-L138) (also noting that ["Resources"](https://github.com/withastro/starlight/blob/main/docs/astro.config.mjs#L162-L169) could be done in the same PR too).

**Pull request merged**